### ISSUE TITLE
USWDS - a11y: Ensure component page landmarks are unique.

### ISSUE THREAD

**Issue by mejiaj** [uswds/uswds-site#1109]
## Description

Related to #1104.

**id:** 'landmark-unique',
**impact:** 'moderate',
**tags:** [ 'cat.semantics', 'best-practice' ],
**help:** 'Ensures landmarks are unique',
**description:** 'Landmarks must have a unique role or role/label/title (i.e. accessible name) combination',
**helpUrl:** 'https://dequeuniversity.com/rules/axe/3.5/landmark-unique?application=axeAPI',

### Pages to fix
- [ ] Sidenav
- [ ] Identifier
- [ ] Footer
- [ ] Breadcrumb
- [ ] Banner

## Steps to reproduce the issue

1. Open `run-axe.js`
2. Add component URLs from #1104 to `const PAGES` array
4. `npm run axe`

**Comment by mejiaj** [comment]
Blocked by #1136

**Comment by thisisdano** [comment]
I'll make sure this is a part of #1136

**Comment by mejiaj** [comment]
Needs re-verification.

**Comment by mejiaj** [comment]
I can't run on my work laptop because of a chrome-driver issue with WSL. @amyleadem or @mahoneycm would you be able to test this?

You need to update `PAGES` in `./config/run-axe.js` to the following:
```js
const PAGES = [
  '/components/',
  '/components/accordion/',
  '/components/alert/',
  '/components/banner/',
  '/components/breadcrumb/',
  '/components/button/',
  '/components/button-groups/',
  '/components/data-visualizations/',
  '/components/footer/',
  '/components/form-controls/',
  '/components/form-templates/',
  '/utilities/layout-grid/',
  '/components/header/',
  '/components/icons/',
  '/components/identifier/',
  '/components/search/',
  '/components/sidenav/',
  '/components/step-indicator/',
  '/components/table/',
  '/components/tag/',
  '/components/tooltip/',
  '/components/typography/',
];
```

And then run `npm run axe`. If the errors don't appear anymore we can close this issue.

**Comment by mahoneycm** [comment]
Found the following errors:

```js
 /components/icons/ Found 1 aXe violations.
[
  {
    id: 'aria-allowed-attr',
    impact: 'serious',
    tags: [ 'cat.aria', 'wcag2a', 'wcag412' ],
    description: "Ensures ARIA attributes are allowed for an element's role",
    help: 'Elements must only use allowed ARIA attributes',
    helpUrl: 'https://dequeuniversity.com/rules/axe/4.4/aria-allowed-attr?application=axeAPI',
    nodes: [
      [Object], [Object], [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object],
      ... 139 more items
    ]
  }
]
```

```js
 /components/ Found 1 aXe violations.
[
  {
    id: 'aria-allowed-role',
    impact: 'minor',
    tags: [ 'cat.aria', 'best-practice' ],
    description: 'Ensures role attribute has an appropriate value for the element',
    help: 'ARIA role should be appropriate for the element',
    helpUrl: 'https://dequeuniversity.com/rules/axe/4.4/aria-allowed-role?application=axeAPI',
    nodes: [
      [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object],
      [Object], [Object], [Object], [Object]
    ]
  }
]
```

**Comment by mahoneycm** [comment]
The `/components/icon` looks like it failed on a whopping 239 tags! (the last failure is seperate)

<img width="885" alt="image" src="https://user-images.githubusercontent.com/25211650/173149724-cae45242-8abc-443a-8cd0-ebda63c41a59.png">



**Fix the following:
aria-label attribute cannot be used on a span with no valid role attribute.**

**Comment by mahoneycm** [comment]
For `/components/` this is actually the same failure as the 240th failure on `/components/icons` so it looks like we'll have to fix them both

<img width="885" alt="image" src="https://user-images.githubusercontent.com/25211650/173150310-9e674381-ed03-4c95-ba21-66878e74f3cd.png">

**Fix the following:
aria-label attribute is not well supported on a div with no valid role attribute.**

**Comment by mejiaj** [comment]
@mahoneycm thanks! The errors look different to the original issue. Could you make a new issue for the a11y errors on Component pages?

Once that's done we can link it here and close this issue.
